docs/integrations/glue_hive_metastore.md

```diff
@@ -7,6 +7,9 @@ redirect_from: /using/glue_hive_metastore.html
 
 # Using lakeFS with the Glue/Hive Metastore
 
+{: .warning }
+**Deprecated Feature:** This feature is being phased out. A superior replacement will be introduced soon.
```

Comment:
These warnings are vague and don't give confidence that we'll continue to support these integrations. 
```suggestion
**Deprecated Feature:** Having heard the feedback from the community, we are planning to replace the below manual steps with an automated process. You can read more about it [here](https://github.com/treeverse/lakeFS/issues/6461).
```

Reply:
Here and elsewhere